float64 first
float64 second
char base
int64 servo1 
int64 servo2
bool isPump